modulus/models/fno/fno.py

```diff
@@ -73,19 +73,25 @@ def __init__(
         self.in_channels = in_channels
         self.num_fno_layers = num_fno_layers
         self.fno_width = fno_layer_size
-        self.coord_features = coord_features
-        # Spectral modes to have weights
-        if isinstance(num_fno_modes, int):
-            num_fno_modes = [num_fno_modes]
+        self.activation_fn = activation_fn
+
         # Add relative coordinate feature
+        self.coord_features = coord_features
         if self.coord_features:
             self.in_channels = self.in_channels + 1
-        self.activation_fn = activation_fn
 
-        self.spconv_layers = nn.ModuleList()
-        self.conv_layers = nn.ModuleList()
+        # Padding values for spectral conv
+        if isinstance(padding, int):
+            padding = [padding]
+        self.pad = padding[:1]
+        self.ipad = [-pad if pad > 0 else None for pad in self.pad]
+        self.padding_type = padding_type
 
-        # Initial lift network
+        # build lift
+        self.build_lift_network()
+        self.build_fno(num_fno_modes)
+
+    def build_lift_network(self) -> None:
```

Comment:
Since these are not private, let's add docstrings to all the `build_lift_network`, `build_fno` etc. methods.

Reply:
done!